src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -43,7 +43,25 @@
  * <pre>
  * &lt;module name=&quot;MissingSwitchDefault&quot;/&gt;
  * </pre>
- *
+ * <p>Example of violation:</p>
+ * <pre>
+ * switch (i) {
+ *  case 1:
+ *    break;
+ *  case 2:
+ *    break;
+ *  // violation
+ * }
+ * </pre>
+ * <p>Example of correct code:</p>
+ * <pre>
+ * switch (i) {
+ *  case 1:
+ *    break;
```

Comment:
Please add case 2 like first example to show when you change bad code to good, violation goes away.

Reply:
Done